### DISCUSSION TITLE
Editable sheet

### DISCUSSION THREAD

**Discussion by drkcide** [JohanFalt/Foundry_WoD20#322] [Q&A] [answered]
I was curious how difficult it would be, if it would even be possible, to be able to build a custom sheet? The system is exactly what my game needs as we use the WoD as a base, but we play in the star wars universe. being able to rename fields for force powers would make things perfect.

**Comment by JohanFalt** [comment]
What fields are we talking about really here?

**Reply by drkcide** [reply]
I've been messing around with the sheet all morning and I think using the base vampire sheet as reference. 
changing Virtue to Force Disciplines so that they could be added to rolls 
Humanity to Force Potential
Conviction/Conscience  to Sense
Self-control- Control
Courage- Alter 
I think this would be a possibility as they change with the humanity path, but not really knowing the underlying difficulty of incorporating something like this.
being able to change Blood Pool to Force points
being able to rename fields on the BIO table to better reflect a star wars sheet. 
Most other things I've found can be customized to be what we needed.

**Comment by JohanFalt** [comment, marked as answer]
So if we are looking at the long run of what is coming:

Conviction, Self-Control and Courage can be what is called Other Traits in the coming version (v2.1) - so that part is covered.
In v2.2 it is mortal time, stuff like Ghouls and stuff and with that I think most think will be handled of what you wish to do... I think. It is a bit off and not very planned out yet so it is hard to tell right out.

Abilities and such is covered in v2.1 as well.
